src/sentry/models/integration.py

```diff
@@ -62,6 +63,53 @@ class Meta:
         unique_together = (("project", "stack_root"),)
 
 
+class ExternalProviders(Enum):
+    GITHUB = 0
+    GITLAB = 1
+
+
+EXTERNAL_PROVIDERS = {
+    ExternalProviders.GITHUB: "github",
+    ExternalProviders.GITLAB: "gitlab",
+}
+
+
+class ExternalTeam(DefaultFieldsModel):
+    __core__ = False
+
+    team = FlexibleForeignKey("sentry.Team")
+    provider = BoundedPositiveIntegerField(
+        choices=(
+            (ExternalProviders.GITHUB, "github"),
+            (ExternalProviders.GITLAB, "gitlab"),
+        ),
+    )
+    external_id = models.CharField(max_length=64)
```

Comment:
Can you add docstring for this column explaining what it contains?
Also I think 64 might be too low.

Reply:
👍  bumped it to a textfield

Reply:
Need to change this to `TextField` as well as in `ExternalUser`